src/test/java/training/article/article.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Create, edit and delete an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+    * def token = "Bearer " + login.response.user.token
+
+  @E2Earticle
+  Scenario: E2E an article
+
+    * def createArticle = call read('create.feature@flow')
+    * def updateArticle = call read('update.feature@flow')
```

Comment:
add a space line and send the data of the created article to the update feature.